### PR TITLE
chore(deps): update dependencies and CI

### PR DESCRIPTION
**Pull request by sambacha**
- Updates dependencies to the correct github location now
- Updates Github Actions Workflow file for LTS NodeJS support and faster CI

### CODE REVIEW

**Comment by sambacha** [comment]
@fubuloubu or @BobTheBuidler ser plz

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -5,14 +5,34 @@ on:
     branches:
     - master
     - develop
+    
   pull_request:
 
+# ensure multiple CI processes are not running analysis on contracts
+concurrency:
```
Comment (fubuloubu) [inline .github/workflows/test.yaml:12]: This is really cool, but could you submit this as a separate PR?

Comment (sambacha) [reply .github/workflows/test.yaml:12]: sure, no problem

**Review by fubuloubu — APPROVED** [review]

**Comment by sambacha** [comment]
btw, is there a reason / way that the contract can be included in the @yfi/sdk nodejs package? seems like it should, and I am guessing you would prefer to not add npm packaging to this repo

**Comment by fubuloubu** [comment]
> is there a reason / way that the contract can be included in the @yfi/sdk nodejs package?

maybe? :man_shrugging: 

> I am guessing you would prefer to not add npm packaging to this repo.

:handshake:
